great_expectations/data_context/data_context.py

```diff
@@ -43,6 +44,23 @@ class DataContext(object):
 
     """
 
+    @classmethod
+    def create(cls, context_root_dir=None):
+        if not os.path.isdir(context_root_dir):
+            logger.error("context_root_dir must a directory in which to initialize a new DataContext")
```

Comment:
"must *be*"

Reply:
Good catch; updated (to raise a DataContextError) with the correct verbiage.